### PR TITLE
fix(dpkg): 1.19.1 has been removed,so use 1.19.2 instead

### PR DESCRIPTION
**Pull request by fsx950223**
Fixes #
dpkg package not found
(let GitHub automatically close an issue when this pull request gets merged)

## Before you submit a pull request
This template is not neccessary when you do simple things like updating packages to the latest version. But in doubt, it's always better so provide some information.

## Description
Provide a description, what your changes do and why they are important
update package version
Please link issues and other pull requests connected to this one.

## Addtional information
Mention things we might need to know. Like:

Works properly:
- [x] x86_64
- [ ] aarch64 (reasons why it doesn't)

---

## That's it
Thank you for submitting your pull request.
When done, please delete the parts of this template which you don't need or these, which are only for guidance.

### CODE REVIEW

File: packages/dpkg.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@ class Dpkg < Package
   description 'A medium-level package manager for Debian'
   homepage 'https://anonscm.debian.org/git/dpkg/'
   version '1.19.1'
```
Comment (uberhacker) [inline packages/dpkg.rb]: This version should be bumped also.

File: packages/dpkg.rb [hunk 2]
```diff
@@ -4,7 +4,7 @@ class Dpkg < Package
...
-  source_url 'http://http.debian.net/debian/pool/main/d/dpkg/dpkg_1.19.1.tar.xz'
+  source_url 'http://http.debian.net/debian/pool/main/d/dpkg/dpkg_1.19.2.tar.xz'
   source_sha256 'ae3978a6b7bddc3e3196804ae0d49ea008c84a8a7a60b7d212af1e1d469e7ccf'
```
Comment (uberhacker) [inline packages/dpkg.rb]: This sha256 hash will change.

**Comment by uberhacker** [comment]
Thank you for the PR @fsx950223!  Please see comments.

File: packages/dpkg.rb (hunk 2 above)
Comment (uberhacker) [reply packages/dpkg.rb]: Please change to:
```
source_url 'https://deb.debian.org/debian/pool/main/d/dpkg/dpkg_1.19.2.tar.xz'
source_sha256 'f8f2ae2cf8065b81239db960b3794099ec607c94a125cec61c986f68f9861b71'
```

File: packages/code.rb [hunk 3]
```diff
@@ -25,7 +26,12 @@ class Code < Package
   depends_on 'libgconf'
   depends_on 'xdg_base'
   depends_on 'sommelier'
-
+  def self.preinstall
+    case ARCH
+    when 'armv7l'
+      system "dpkg", "-i", "vscode-1.28.0.deb"
+    end
+  end
```
Comment (uberhacker) [inline packages/code.rb]: Please include the change to code.rb in a separate PR.

Comment (fsx950223) [reply packages/code.rb]: ok

File: packages/code.rb [hunk 4]
```diff
@@ -17,6 +17,7 @@ class Code < Package
   end
 
   binary_url ({
+    armv7l:'https://github.com/stevedesmond-ca/vscode-arm/releases/download/1.28.0/vscode-1.28.0.deb'
```
Comment (uberhacker) [inline packages/code.rb]: I don't think you will want to this with a .deb package.  This is reserved for Chromebrew packages only.

**Pull request merged**